agent/structs/config_entry_gateways.go

```diff
@@ -173,7 +173,7 @@ func (e *IngressGatewayConfigEntry) Normalize() error {
 	}
 
 	e.Kind = IngressGateway
-	e.EnterpriseMeta.Normalize()
+	e.Normalize()
```

Comment:
So, pretty sure this needs to switch back, otherwise you're in an infinite call loop:

```suggestion
	e.EnterpriseMeta.Normalize()
```

Reply:
oh yup, good catch